### ISSUE TITLE
ENH: add an optional check to guarantee that all groupped accessions come from the same study

### ISSUE THREAD

**Issue by yarikoptic** [dartmouth-pbs/heudiconv#21]
ATM, whenever someone sends me a list of his accessions, I need first to check manually if all of them actually correspond to that study and do not come from someone else's.  But that information is readily available after sorting, so we could just quickly check and fail if there is some oddity such as 
- different "locator" value
- some have session defined, some not
- theoretically (might be difficult since completion is ) we could even do check/fail if produced filenames for the do not conform to BIDS

**Comment by mvdoc** [comment]
But theoretically we'll never have to run heudiconv with grouped accession numbers anymore, right? we'll be running heudiconv separately for different accession numbers with the monitor, or am I missing something?

**Comment by yarikoptic** [comment]
You are correct. Just thought that until then we might want this feature which is easy to add
